src/main/java/core/basesyntax/ColorList.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+public enum ColorList {
+        Green,
+        Red,
+        Black,
+        Blue,
+        Lilac,
+        Turquoise,
+        Yellow,
+        NavyBlue
+}
```

Comment:
в enum элементы списка пишутся с большой буквы
```suggestion
public enum ColorList {
        GREEN,
        RED,
```

и так далее